README.rst

```diff
@@ -20,21 +20,25 @@
 .. _Depsy: http://depsy.org/package/python/matplotlib
 
 ##########
-matplotlib
+Matplotlib
 ##########
 
-matplotlib is a Python 2D plotting library which produces publication-quality
-figures in a variety of hardcopy formats and interactive
-environments across platforms. matplotlib can be used in Python
-scripts, the Python and IPython shell (ala MATLAB or Mathematica), web
-application servers, and various graphical user interface toolkits.
+Matplotlib is a Python 2D plotting library which produces publication-quality
+figures in a variety of hardcopy formats and interactive environments across
+platforms. Matplotlib can be used in Python scripts, the Python and IPython
+shell (ala MATLAB or Mathematica), web application servers, and various
```

Comment:
@QuLogic is going to do a pass over it, but it might be easier for me to fix this one considering my awesome french keyboard (Yéǎh¡!).
I'll do another pass tonight to check for this particular point )